src/Datum.ts

```diff
@@ -194,7 +199,7 @@ export const getEq = <A>(E: Eq<A>): Eq<Datum<A>> => ({
  *
  * @since 2.0.0
  */
-export const getSemigroup = <A>(S: Semigroup<A>): Semigroup<Datum<A>> => ({
+export const getSemigroup = <A>(S: semigroup.Semigroup<A>): semigroup.Semigroup<Datum<A>> => ({
```

Comment:
I thinks we can add a deprecated tag to this as I would like to replace it with your getProgressSemigroup in 4.0.